### PR TITLE
refactor: separate click wrappers from core evaluation functionality <- Ingest test fixtures update

### PR DESCRIPTION
**Pull request by ryannikolaidis**
This pull request includes updated ingest test fixtures.
Please review and merge if appropriate.

### CODE REVIEW

File: test_unstructured_ingest/metrics/all-docs-cct.tsv [hunk 1]
```diff
@@ -1,3 +1,3 @@
-filename	connector	cct-accuracy	cct-%missing
```
Comment (shreyanid) [inline test_unstructured_ingest/metrics/all-docs-cct.tsv:1]: not a problem, added doctype column and (for some reason) switched order of documents. scores unchanged

File: test_unstructured_ingest/metrics/aggregate-scores-cct.tsv [hunk 2]
```diff
@@ -1,3 +1,3 @@
 strategy	average	sample_sd	population_sd	count
-cct-accuracy	0.798	0.083	0.072	4
```
Comment (shreyanid) [inline test_unstructured_ingest/metrics/aggregate-scores-cct.tsv:2]: @Klaijan I didn't change anything about the calculation of these values in my click wrapper refactor PR - is this change expected / acceptable?

Comment (Klaijan) [reply test_unstructured_ingest/metrics/aggregate-scores-cct.tsv:2]: Yep should be ok! I updated the connectors (where to and not to copy files from) on Saturday and potentially did not run the fixture update.

Comment (shreyanid) [reply test_unstructured_ingest/metrics/aggregate-scores-cct.tsv:2]: ok thanks for the update!

**Review by shreyanid — APPROVED** [review]

**Pull request merged**